VAMobile/src/screens/HomeScreen/VeteransCrisisLineScreen/VeteransCrisisLineNumbers/VeteransCrisisLineNumbers.tsx

```diff
@@ -28,12 +28,11 @@ const VeteransCrisisLineNumbers: FC = () => {
   return (
     <>
       <Box>
-        <ClickForActionLink
-          testID="veteransCrisisLineCrisisCallNumberTestID"
-          displayedText={t('veteransCrisisLine.crisisCallNumberDisplayed')}
+        <ClickToCallPhoneNumber
+          ttyBypass
+          phone={t('988')}
           a11yLabel={t('veteransCrisisLine.crisisCallNumberDisplayed.a11y')}
-          numberOrUrlLink={t('988')}
-          linkType={LinkTypeOptionsConstants.call}
+          displayedText={t('veteransCrisisLine.crisisCallNumberDisplayed')}
           fireAnalytic={fireAnalyticFn}
```

Comment:
One more spot that you can update to ClickToCallPhoneNumber below for the TTY one

Reply:
Because that's the tty number, I didn't change it, because it will no longer display the correct information if it uses the ClickToCallPhoneNumber component. With the new component the icon is just the phone (not the tty icon), and it'll just add the `711` number to it, which from what I understand is not correct, so I left it as the old component with the `LinkTypeOptionsConstants.callTTY` (this PR only replaced the `LinkTypeOptionsConstants.call` link types).